share/functions/ip.fish

```diff
@@ -0,0 +1,9 @@
+#
+# Enable ip colored output if this is available
+#
+
+if command ip -c l >/dev/null 2>/dev/null
+    function ip
+        command ip -c $argv
```

Comment:
This enables color _always_, even if the output isn't to a terminal.

You want `-color=auto` instead (yes, one dash).

Reply:
> Option "-color=auto" is unknown, try "ip -help".
Do you really have this flag for `ip`? Probably I need to do it like @floam suggested with `status is_interactive`

Reply:
>Probably I need to do it like @floam suggested with status is_interactive

No. That does not apply. The issue is **not** "is fish interactive", the issue is "is this output going to the terminal"? Which `isatty stdout` would be okay for.

Which `ip` is this? `ip -V`?

Reply:
Good, thanks!

```
> ip -V
ip utility, iproute2-ss180813
```

Reply:
Ah, man.... yeah, that was probably added recently. My version is

```
ip utility, iproute2-ss181023
```

I have no idea how the versions work here, but that doesn't seem to be much of a difference.

Reply:
Oh, and I thought my Fedora is bleeding edge :stuck_out_tongue: 

Do you want me to change it to use the newer flag? But I can't test it then.

Reply:
>Do you want me to change it to use the newer flag?

No, we don't want to force everyone to update ip. In general with the features we use from elsewhere we want to be a bit conservative, to give everyone a chance to catch up.